### PR TITLE
pass-git-helper: 1.1.0 -> 1.1.1

### PR DESCRIPTION
**Pull request by languitar**
###### Motivation for this change

I have released a new version upstream of the package. As I don't use nix and have no tools available, I don't know how to compute and updated sha256sum for this. If someone could help, that would be great.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/version-management/git-and-tools/pass-git-helper/default.nix [hunk 1]
```diff
@@ -2,13 +2,13 @@
...
   pname   = "pass-git-helper";
-  version = "1.1.0";
+  version = "1.1.1";
 
   src = fetchFromGitHub {
     owner  = "languitar";
     repo   = "pass-git-helper";
     rev    = "v${version}";
-    sha256 = "18nvwlp0w4aqj268wly60rnjzqw2d8jl0hbs6bkwp3hpzzz5g6yd";
+    sha256 = "TODO";
```
Comment (lourkeur) [inline pkgs/applications/version-management/git-and-tools/pass-git-helper/default.nix:11]:
```suggestion
    sha256 = "sha256-GdsFPpBdoEaOCmdKxw5xTuFOcGFH94w5q/lV891lCUs=";
```

Here you go.

**Comment by lourkeur** [comment]
NVM here's something that builds afaict:
1d3fd9fc6f437bfe7df65145dedfce859884c79c

**Comment by languitar** [comment]
Sorry for the delay and thanks for the help. I have updated the commit.

File: pkgs/applications/version-management/git-and-tools/pass-git-helper/default.nix [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
     owner  = "languitar";
     repo   = "pass-git-helper";
     rev    = "v${version}";
-    sha256 = "18nvwlp0w4aqj268wly60rnjzqw2d8jl0hbs6bkwp3hpzzz5g6yd";
+    sha256 = "sha256-GdsFPpBdoEaOCmdKxw5xTuFOcGFH94w5q/lV891lCUs=";
   };
 
   propagatedBuildInputs = [ pyxdg ];
-  checkInputs = [ pytest ];
+  checkInputs = [ pytest, pytest-mock ];
```
Comment (lourkeur) [inline pkgs/applications/version-management/git-and-tools/pass-git-helper/default.nix:15]:
```suggestion
  checkInputs = [ pytest pytest-mock ];
```
As you can tell from the CI, lists are not comma-separated in Nix.

**Review by bbjubjub2494 — CHANGES_REQUESTED** [review]

**Comment by lourkeur** [comment]
The `stdenv.lib` refactor caused a merge conflict. Can you rebase or use 973a0d3ddbc334969f318941f202f40d00643ad1?

**Comment by languitar** [comment]
Rebased

**Review by bbjubjub2494 — APPROVED** [review]
- [x] Tested compilation of all pkgs that depend on this change using `nixpkgs-review`
```
1 package built:
pass-git-helper
```

- [x] Tested execution of all binary files (one)

File: pkgs/applications/version-management/git-and-tools/pass-git-helper/default.nix [hunk 3]
```diff
@@ -1,18 +1,18 @@
...
     owner  = "languitar";
     repo   = "pass-git-helper";
     rev    = "v${version}";
-    sha256 = "18nvwlp0w4aqj268wly60rnjzqw2d8jl0hbs6bkwp3hpzzz5g6yd";
+    sha256 = "sha256-GdsFPpBdoEaOCmdKxw5xTuFOcGFH94w5q/lV891lCUs=";
   };
 
   propagatedBuildInputs = [ pyxdg ];
-  checkInputs = [ pytest ];
+  checkInputs = [ pytest pytest-mock ];
```
Comment (SuperSandro2000) [inline pkgs/applications/version-management/git-and-tools/pass-git-helper/default.nix:15]: It might be worth it to move pytestCheckHook.

**Pull request merged**